frontend/src/routes/Provider/components/DocsMenu.tsx

```diff
@@ -107,11 +117,23 @@ export function ProviderDocsMenu() {
   const { data } = useSuspenseQuery(
     getProviderVersionDataQuery(namespace, provider, version),
   );
-
+  const [searchFilter, setSearchFilter] = useState("");
+  const filterInput = (
+    <input
+      type="filter"
+      placeholder="Filter..."
+      className="mb-2 px-4 py-2 text-sm text-gray-500 dark:text-gray-400"
```

Comment:
We can reuse styles from the main search bar which supports the focus state:
```suggestion
      className="mb-2 h-9 w-full appearance-none border border-transparent bg-gray-200 px-4 text-inherit placeholder:text-gray-500 focus:border-brand-700 focus:outline-none dark:bg-gray-800"
```